gamestonk_terminal/feature_flags.py

```diff
@@ -19,9 +19,6 @@
 # Enable Prediction features
 ENABLE_PREDICT = strtobool(os.getenv("GTFF_ENABLE_PREDICT", "False"))
 
-# Enable FB Prophet
```

Comment:
Is this because prophet was dropped all along?

Reply:
I think I dropped it here based off a conversation in a different thread.

Reply:
Yup, I read that somewhere. Was happy about that, ngl :D